grpc/cubic.proto

```diff
@@ -276,8 +273,25 @@ message RecognitionResult {
 // A recognition hypothesis
 message RecognitionAlternative {
   // Text representing the transcription of the words that the user spoke.
+  //
+  // The transcript will be formatted according to the servers formatting
+  // configuration. If you want the raw transcript, please see the field
+  // `raw_transcript`.
```

Comment:
Should we say something about how if you *only* want the raw transcript, you should just configure no formatting? Since this field will always be populated, we don't want the redundancy of people specifying `enable_raw_transcript=true` and having the same text included twice.